userguide/tutorials/metrics-newrelic.adoc

```diff
@@ -3,56 +3,23 @@
-every conceivable view of your system's behavior.
-
-Several third-party tools are available to manage and display metrics of
-this type. One popular tool is https://www.newrelic.com/[New Relic]. This
+Several third-party tools are available to manage and display the metrics produced by Kill Bill. One popular tool is https://www.newrelic.com/[New Relic]. This
 package offers a low-cost option to monitor and display whatever metrics
 you may find useful from the Kill Bill collection, including metrics
 that show the performance and resource use of Kill Bill and Kaui
 themselves, and metrics describing the infrastructure on which these
 systems run.
 
 New Relic can be integrated with several types of Kill Bill installations. In this guide we discuss the use of New Relic to manage metrics produced
-by two types of implementations on Amazon Web Services (AWS):
-single-tier and multi-tier. With the latest Amazon Machine Image (AMI)
-for Kill Bill, it is very easy to direct the Kill Bill metrics to
-New Relic and work with them in the way that best suits your needs.
-
-== Check Your Kill Bill Version
-
-We assume that you have a working Kill Bill implementation running on
-AWS, using either the
-https://docs.killbill.io/latest/aws-singletier.html[Single Tier] or
-https://docs.killbill.io/latest/aws-multitier.html[Multitier]
-installation options. The single-tier option runs Kill Bill, Kaui, the
-MariaDB database, and the nginx load balancer on a single AWS EC2
-instance. This is suitable for quick deployment and for testing and
-development. The multi-tier option runs Kill Bill and Kaui on two or more
-separate EC2 instances, and uses an AWS RDS database manager along with
-an AWS load balancer. This is an attractive option for many production
-uses.
-
-You should be running Kill Bill version 2021.9 or later. This version
-can be configured to provide its full collection of Kill Bill metrics to
-New Relic (and several other analysis tools). If you open the Kill Bill
-admin page in your browser, it should show that you are
-running the "Kill Bill Premium edition":
-
-image::metrics-kb-admin.png[align=center]
-
+by single-tier and multi-tier implementations on Amazon Web Services (AWS). The latest Amazon Machine Image (AMI)
+for Kill Bill is preconfigured to direct the Kill Bill metrics to
+New Relic and work with them in the way that best suits your needs. These instructions apply to the Kill Bill AMI version 2021.9 or later.
 
-If you see this page you are ready to go. If not, you should upgrade
-your installation.
-
-While you are on this page, if you want to see all the metrics that Kill
-Bill has to offer, just add the path `/1.0/metrics` to the URL. Have fun!
-If you find these hard to understand, though, maybe New Relic can help.
+To set up New Relic to manage the metrics available from Kill Bill, there
+are just two steps: (1) setup a New Relic account, and (2)
+edit a configuration file to enable the transfer of the complete Kill Bill metrics.
```

Comment:
Maybe split the steps into two separate lines to improve readability

Reply:
Done. I was debating whether this should be a list.